app/_src/policies/meshexternalservice.md

```diff
@@ -0,0 +1,317 @@
+---
+title: MeshExternalService
+---
+
+{% warning %}
+Do **not** combine with [External Service](/docs/{{ page.version }}/policies/external-services).
+{% endwarning %}
+
+This resource allows services running inside the mesh to consume services that are not part of the mesh.
+The `MeshExternalService` resource allows you to declare external resources instead of relying on [MeshPassthrough](/docs/{{ page.version }}/policies/meshpassthrough) or [passthrough mode](/docs/{{ page.version }}/networking/non-mesh-traffic#outgoing).
+In contrast to passthrough `MeshExternalService` behaves like a normal service as if it was part of the mesh.
```

Comment:
as a first-time reader it's still not clear what's the difference and what it means that it's "normal service".

I would try to rephrase it so that mesh passthrough simply allows traffic to flow while MeshExternalService allows us to use policies like MeshTrafficPermission, MeshRateLimit etc.

Reply:
I removed that sentence because targeting does not work yet (I thought on egress it did) and added a note that the resource is experimental.